src/main/java/org/opensearch/flowframework/transport/handler/WorkflowFunction.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.transport.handler;
+
+/**
+ * Interface for a workflow function.
+ *
+ */
+public interface WorkflowFunction {
+    /**
+     * Performs this operation.
+     *
+     * Notes: don't forget to send back responses via channel if you process response with this method.
```

Comment:
This isn't clear what it means. Which channel since it's not passed as a parameter?  Should this be a `Consumer<Channel>`?

Reply:
Removed the interface itself